src/Kernel/Number.class.st

```diff
@@ -707,19 +738,30 @@ Number >> roundUpTo: aNumber [
 { #category : #'truncation and round off' }
 Number >> rounded [
 	"Answer the integer nearest the receiver."
+	
+	"1.4 rounded >>> 1"
+	"1.5 rounded >>> 2"
+	"2 rounded >>> 2"
+	"-1.5 rounded >>> -2"
```

Comment:
Just thoughts for a future PR:
the comment should tell that this method always round AWAY from zero in case of tie

Then provide an example of tie with even integer part `2.5 rounded >>> 3`
That is because some language might use other convention (round tie to nearest even).